### ISSUE TITLE
Add new labels to reflect the state of a PR

### ISSUE THREAD

**Issue by vishh** [kubernetes/kubernetes#30034]
PRs either need attention from reviewer(s) or author(s). It will be helpful for both authors and reviewers if the state of the PR is explicit. 

As of today, it is difficult to say if a PR has been reviewed or if the author has addressed all comments.

One way to address this will be for reviewers to close PRs that need attention from authors before being merged. This approach has the downside of running all tests even on trivial PRs and might also send a wrong message to new contributors.

Another option is to add labels - e.x. `needs-review` & `needs-update`. 
Given that we have bot automation, we can let all PR authors automatically apply `needs-review` label by posting a `needs-review` message in the PR.
The same bot logic can be used to apply `needs-update` label too.

This will help reviewers ignore PRs that haven't been updated since the last review. 

cc @bgrant0607 @grod

**Comment by bgrant0607** [comment]
@vishh This is the plan.

Workflow description in progress:
https://docs.google.com/document/d/194MGgJHlm4ovXtvzvkjzzBZG_v47d95nsjuuaUe1Log/edit?ts=57929ed1#heading=h.h8u91v47whwa

Info about the PR dashboard:
https://groups.google.com/d/msgid/kubernetes-wg-contribex/CABKCOGkqSaJ99WsKWr1YnTdE7qNQnsvNOUc8Zd%2Bctq2VCzp9LQ%40mail.gmail.com

Will add this to the roadmap, if it's not a pure dupe:
https://github.com/kubernetes/community/wiki/Roadmap:-Contributor-Experience

cc @apelisse @rmmh @fejta @pwittrock

**Comment by asalkeld** [comment]
how does reviewable fit into this?

**Comment by grodrigues3** [comment]
https://github.com/kubernetes/contrib/pull/1582  This PR is intended to parse human commands as well as nag the person (author or reviewer) whose action is required.

**Comment by vishh** [comment]
@grodrigues3 What will be the trigger phrases to switch ownership of PRs between authors and assignees?

**Comment by apelisse** [comment]
We are not completely there yet, we would love a suggestion :-)

**Comment by vishh** [comment]
How about PR author says "ready for review" & assignee(s) says "needs more work"

**Comment by apelisse** [comment]
The idea is that we are going to go for "commands", in the IRC/slack sense:

Ready for review: `/review`
Needs more work: `/needs-work` ?

**Comment by vishh** [comment]
That's even better. sgtm! How about `/needs-review` instead of `/review`?

**Comment by apelisse** [comment]
Yeah that makes sense. We should try to get the minimum amount to type though. We should plan to have a list of all the use-cases and make sure everyone agrees on the commands!
